### ISSUE TITLE
losses

### ISSUE THREAD

**Issue by ghost** [cvlab-yonsei/PISR#5]
Hi

Thanks for your code.

I have another question about loss function.

loss in the paper is written as :  

![스크린샷 2020-09-05 오후 10 53 03](https://user-images.githubusercontent.com/66544563/92306607-54e64c80-efcb-11ea-806e-20b9a1c41a8b.png)

but I see the loss in the code as : 
![스크린샷 2020-09-05 오후 10 52 37](https://user-images.githubusercontent.com/66544563/92306614-6b8ca380-efcb-11ea-8f97-241d52eca5d3.png)

I wonder if these two equations are the same. 

"torch.log(2*std) + numerator / (std)" this part is almost the same with one in the paper
but why "mu.shape[1] * np.log(2*math.pi)/2" term exists?

sorry for many questions but It'll save a lot of my time if you answer me ;) 
thanks in advance.

**Comment by wonkyunglee** [comment]
Hello, 
The term "mu.shape[1] * np.log(2math.pi)/2" is just a constant since it doesn't contain any parameters to update. 
In fact, the log(2*pi) came from the log-likelihood function of the gaussian distribution, but we didn't remove it by mistake. 
However, I'm sure that removing the term doesn't affect performance.

**Comment by ghost** [comment]
wow!

your reply helped me a lot!

thanks!

**Comment by star-lunar** [comment]
> Hi
>Have you adjusted the author's code? After training the teacher network,I got the indications :best psnr=0.000 
> I don't know why
> I would appreciate it if you could give me a reply

**Comment by Orange066** [comment]
HI!
 Why std * 0.1? Why 0.1?
